src/jit/emitxarch.cpp

```diff
@@ -3483,7 +3483,7 @@ void emitter::emitIns_R_I(instruction ins, emitAttr attr, regNumber reg, ssize_t
         sz += emitGetRexPrefixSize(ins);
     }
 
-#ifdef _TARGET_X86_
+#if defined(_TARGET_X86_) && defined(LEGACY_BACKEND)
```

Comment:
>#if defined(_TARGET_X86_) [](start = 0, length = 25)

We have AVX emitter functionality under _TARGET_AMD64_.  You might want to have a separate git issue for enabling AVX2 support by default.